.github/workflows/build-container.yml

```diff
@@ -36,7 +36,7 @@ jobs:
         REPO: ${{ matrix.repo }}
         TIMESTAMP: ${{ needs.make-date-tag.outputs.dtag }}
       run: |
-        docker_repo=opensciencegrid/osgvo-docker-pilot
+        docker_repo=${GITHUB_REPOSITORY/opensciencegrid\/docker-/opensciencegrid/}
```

Comment:
tbh the hard-coded name was intentional here since the github/dockerhub repo names to not follow the normal docker-XYZ pattern.  This recipe that strips the leading `docker-` component works, but in this case only because the github and dockerhub repo names are identical.

Reply:
Yup, that's fine by me and I've been thinking that we should rename this repo to `docker-backfll-container` or something like that such that the image becomes `opensciencegrid/backfill-container`